src/aggregate/aggregate_exec.c

```diff
@@ -38,13 +39,21 @@ static size_t serializeResult(AREQ *req, RedisModuleCtx *outctx, const SearchRes
   }
 
   if (options & QEXEC_F_SEND_SCORES) {
-    RedisModule_ReplyWithDouble(outctx, r->score);
+    if ((options & QEXEC_F_SEND_SCORE_EXPLAIN) == false) {
```

Comment:
`if (!(options & QEXEC_F_SEND_SCORE_EXPLAIN))`

Its not that important but this is the styling we use in redisearch

Reply:
:+1: